crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -40,6 +41,29 @@ use std::{
 /// For more information on these consult the item's corresponding documentation.
 ///
 /// [`Or`]: crate::query::Or
+///
+/// # Derive
+///
+/// This trait can be derived with the [`derive@super::WorldQuery`] macro.
```

Comment:
```suggestion
/// This trait can be derived with the [`derive@super::WorldQuery`] macro.
/// To do so, all fields in the struct must themselves impl [`WorldQuery`].
///
/// The fields can be unpacked in the order they are defined by the struct.
```

Reply:
Thank you. One little question: should we include the comment about unpacking order? It seems to be unnecessary, since the values can be unpacked in pretty much any order

Reply:
Oh interesting; I'd forgotten about that. Yeah, feel free to cut the unpacking order comment then.